src/components/customizableAside.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import CustomizableAvatar from "../components/common/customavatar/CustomizableAvatar";
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar"
+
 
 function CustomizableAside({asideCustomClass, src, height, width, alt, headerText, paragraphText}) {
```

Comment:
@Sheyitofunmi We have this component already right? I think you can just import.